src/android/IntercomBridge.java

```diff
@@ -63,6 +62,9 @@ private void setUpIntercom() {
                     String appId = IntercomBridge.this.preferences.getString("intercom-app-id", bundle.getString("intercom_app_id"));
 
                     Intercom.initialize(IntercomBridge.this.cordova.getActivity().getApplication(), apiKey, appId);
+                    if (shouldPing && Bridge.getApi() != null) {
```

Comment:
This all looks good, I wonder if in the future we should try and keep track of if ping happened at all. My thinking is if we hit here and shouldPing is true but Bridge.getApi() is null I wonder could Intercom have knowledge of that and on next request fire a ping too. Not for this PR either way.